src/wallet/rpcwallet.cpp

```diff
@@ -2010,6 +2054,18 @@ UniValue gettransaction(const JSONRPCRequest& request)
             "    }\n"
             "    ,...\n"
             "  ],\n"
+            "  \"asset_details\" : [\n"
+            "    {\n"
+            "      \"asset_type\" : \"new_asset|transfer_asset|reissue_asset\", (string) The type of asset transaction\n"
+            "      \"asset_name\" : \"asset_name\",          (string) The name of the asset\n"
+            "      \"amount\" : x.xxx,                 (numeric) The amount in " + CURRENCY_UNIT + "\n"
+            "      \"address\" : \"address\",          (string) The raven address involved in the transaction\n"
+            "      \"vout\" : n,                       (numeric) the vout value\n"
+            "      \"category\" : \"send|receive\",    (string) The category, either 'send' or 'receive'\n"
```

Comment:
`receive` here is inconsistent with `received` above.  (also change to `sent` here too if you changed it above)

Reply:
Typo, will fix